internal/http/services/owncloud/ocgraph/conversions.go

```diff
@@ -161,13 +161,9 @@ func (s *svc) buildGrantedToForOCMShare(ctx context.Context, grantee *provider.G
 
 	switch grantee.Type {
 	case provider.GranteeType_GRANTEE_TYPE_USER:
-		u, err := s.getUserInfo(ctx, grantee.GetUserId())
-		if err != nil {
-			return nil, errors.New("Failed to fetch user info")
-		}
 		grantedTo.SetUser(libregraph.Identity{
 			Id:          libregraph.PtrString(grantee.GetUserId().OpaqueId),
-			DisplayName: u.DisplayName,
+			DisplayName: grantee.GetUserId().OpaqueId + "@" + grantee.GetUserId().Idp,
```

Comment:
What happens if you do this with CERN users? Wont they show up as `lopresti@` if the IdP is empty? Or at least show up as `lopresti@cern.ch` (which might be fine)?